src/test_c2s_srv.c

```diff
@@ -44,13 +46,14 @@ int raw_read(Connection *conn, char* buff, size_t buff_size,
     // TODO: Keep track of the number of SSL renegotiations that occur
     ERR_clear_error();
     *bytes_read = SSL_read(conn->ssl, buff, buff_size);
+    ssl_errno = errno;
     if (*bytes_read <= 0) {
-      ssl_errno = errno;
       ssl_error = SSL_get_error(conn->ssl, *bytes_read);
       if (is_recoverable_ssl_error(ssl_error, ssl_errno)) {
         return 0;
       } else {
-        log_println(1, "Unrecoverable SSL error in C2S test");
+        log_println(1, "Unrecoverable SSL error in C2S test: %s (%d, errno=%d)",
```

Comment:
Thank you. Would you be willing to include the SSL operation (`SSL_read` in this case) in the error message? This is true for the earlier messages we added in previous PRs (or adjacent log lines).